src/main/java/redis/clients/jedis/JedisSentinelPool.java

```diff
@@ -265,7 +286,7 @@ private HostAndPort initSentinels(Set<String> sentinels, final String masterName
     return master;
   }
 
-  private HostAndPort toHostAndPort(List<String> getMasterAddrByNameResult) {
+  protected HostAndPort toHostAndPort(List<String> getMasterAddrByNameResult) {
```

Comment:
Why change to `protected`?

Reply:
I can't say why the original author made the change. However it does seem unnecessary so I have reverted it back to private.

Reply:
Oh, okay.